cluster/addons/fluentd-gcp/fluentd-gcp-configmap.yaml

```diff
@@ -421,8 +423,7 @@ data:
       # Use multiple threads for processing.
       num_threads 2
       use_grpc true
-      # Use Metadata Agent to get monitored resource.
-      enable_metadata_agent true
+      adjust_invalid_timestamps false
```

Comment:
Some comment?

```
      # Skip timestamp adjustment as this is in a controlled environment with
      # known timestamp format. This helps with CPU usage.
```

Reply:
We should apply this to `fluentd-gcp-configmap-old.yaml` as well.

Reply:
Applied the comment only to the new configs, same question for old configs as mentioned above.

Reply:
Sync'ed offline. Let's leave the old config as-is for now.